hh-dep-monitoring-service/src/main/java/ru/hh/school/depmonitoring/dao/AbstractDao.java

```diff
@@ -0,0 +1,57 @@
+package ru.hh.school.depmonitoring.dao;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.inject.Inject;
+import java.io.Serializable;
+import java.util.List;
+
+public class AbstractDao<T extends Serializable> implements Dao<T> {
+
+    private Class<T> clazz;
+
+    @Inject
+    SessionFactory sessionFactory;
```

Comment:
Если тут делать SessionFactory, то:
```suggestion
    protected final SessionFactory sessionFactory;
```
и убирать из наследников

Reply:
ok